charts/shoot-core/components/charts/network-policies/templates/metrics-server-from-vpn.yaml

```diff
@@ -0,0 +1,21 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  annotations:
+    gardener.cloud/description: |
+      Allows Igress for metrics-server from vpn
+  name: gardener.cloud--allow-metrics-server-from-vpn
+  namespace: kube-system
+  labels:
+    origin: gardener
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: metrics-server
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - podSelector:
+        matchLabels:
+          app: vpn-shoot
```

Comment:
There is a policy which already exists - https://github.com/gardener/gardener/blob/4dab9070a5ece73b2a6ad1a585a036c4a410108b/charts/shoot-core/components/charts/network-policies/templates/allow-from-seed.yaml#L1-L24

Label applicable pods with `networking.gardener.cloud/from-seed: allowed` instead